xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-message/src/main/resources/ApplicationResources.properties

```diff
@@ -0,0 +1,48 @@
+# ---------------------------------------------------------------------------
```

Comment:
Should this file be removed ? All those translations are already provided by xwiki-rendering-macro-message.

Reply:
Addressed in https://github.com/xwiki/xwiki-platform/pull/3023/commits/2df9eece0f9c674dc55d438e2de08637d387e01a :+1: